engine/src/main/java/org/terasology/logic/console/commands/ClientCommands.java

```diff
@@ -68,6 +68,12 @@ public String setWorldTime(@CommandParam("day") float day) {
         return "World time changed";
     }
 
+    @Command(shortDescription = "Permanently sets the current world time in days", requiredPermission = PermissionManager.CHEAT_PERMISSION)
+    public  String setPermanentWorldTime(@CommandParam("day") float day) {
+        worldProvider.getTime().togglePermanentTime();
+        worldProvider.getTime().setDays(day);
+        return "World time changed";
```

Comment:
Might want to change the return so that it more accurately represents the command. "World time change" is generic as can be seen in the command above also using the exact same message.

Reply:
Sure! I will make the required changes 👍

Reply:
Much nicer!